content/post/le-catalogage-setoffe.en.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "New cataloguing developments"
+date: 2020-07-06T08:23:38+02:00
+publishdate: 2020-07-06
+draft: false
+tags: ["rero-ils", "release", "acquisition", "display", "metadata", "predictions"]
+slug: le-catalogage-setoffe
+---
+
+A new sprint ends for {{< smallcaps "rero ils" >}} with a version full of new features. Thanks to an increasingly stable technical foundation, the development team can now focus on user-oriented functionality. The full release notes are available as usual on Github: [`v0.10.0`][1].
+
+<!--more-->
+
+## Interface improvements
+
+UI work on the public and professional interfaces continues. This version brings many adjustments, and in particular a redesign of the metadata editor (cataloguing interface) to improve its readability. The editor is now displayed across the full width of the screen and the fields are easier to distinguish visually with the titles of the main fields now in bold and some sub-fields that can be displayed inline.
+
+{{< figure src="/img/rero-ils-editeur.png" caption="Displaying sub-fields inline instead of in blocks and field titles in bold">}}
+
+## Importing records
+
+Being able to import records from external catalogues streamlines much of the cataloguing work for librarians. {{< smallcaps "rero ils" >}} offers the possibility to do so via [the SRU protocol][2]. The librarian can search for the desired record in the selected catalogue (currently only BnF) using a simple query (keywords for the author, title, date, ISBN…). They can choose the record that they want to import. This record can then be freely edited in the cataloguing interface before being saved.
+
+## New feature: receive an issue
+
+Another brand new feature of the `v0.10.0` release is the journal arrivals module. The holding's **frequency** field, based on [RDA standards][3], enables the system to automatically predict the dates of the next regular issues and to receive them with one click. It is of course also possible to create an item for an irregular issue.
```

Comment:
possible improvement for :`It is of course also possible to create an item for an irregular issue.`
to something like:
`It is of course also possible to receive an irregular issue.
 System creates automatically an item for all received issues.`

Reply:
Done.